### ISSUE TITLE
BGC0000894 (citrinin) is missing genes and should be a Polyketide

### ISSUE THREAD

**Issue by althonos** [mibig-secmet/mibig-json#82]
**Affected BGC: [BGC0000894](https://mibig.secondarymetabolites.org/repository/BGC0000894/index.html#r1c1)**

## Genes

BGC0000894 is missing many gene annotations, in particular the `pksCT`. Below you can see the MIBiG annotations against the genes from the paper:

![mibig](https://user-images.githubusercontent.com/8660647/101113504-cdb36f80-35df-11eb-92d4-3370c5c6eb58.png)

![orfs](https://user-images.githubusercontent.com/8660647/101113509-d015c980-35df-11eb-9db2-81e84b404f13.png)

## Class

BGC0000894 (like its homolog [BGC0001388](https://mibig.secondarymetabolites.org/repository/BGC0001338/index.html#r1c1)) produces citrinin, which is a Polyketide. This cluster should be added to the *Polyketide (Other)* class instead of the *Other* class like it is now.

**Comment by althonos** [comment]
Isn't it possible to add additional annotations using the `extra_genes` defined in the JSON schema, to specify the location of genes that are not in the GenBank record?

**Comment by zdk123** [comment]
Or fix the ORFs manually and include in the genbank subdirectory

**Comment by kblin** [comment]
Yes, both options would work. It's more of a question of how much time we can dedicate to fixing a single issue.
